app/controllers/conversations_controller.rb

```diff
@@ -19,8 +19,16 @@ def index
     render json: current_user.conversations
   end
 
+  def show
+    render json: conversation
```

Comment:
Are you using json builder? If not check the documentation for `jbuilder` gem to generate jsons for this endpoints. If you are a json builder view in the views folder under `controllers` path called `show.json.jbuilder`, it will be rendered automatically, no need to render manually, it will look for a file under that path

Reply:
It's a rails api only app.  So we have no views in here. 
And this endpoint already responds a json (thanks to active model serializers)